src/proto/callbacks.proto

```diff
@@ -0,0 +1,347 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Would it be better to rename this file `lbann_callbacks.proto` to avoid name collisions?

Reply:
I asked @ndryden and he suggested `callbacks.proto`. I'd say "there's not provably a problem until there's provably a problem." and changing it would be premature pessimization.

Reply:
Fair.